plugins/modules/k8s_log.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Fabian von Feilitzsch <@fabianvf>
```

Comment:
Not sure if this is the license block you want.

Reply:
I think it's in keeping with the way the other modules handle copyright, what would you suggest instead?

Reply:
Had been expecting https://github.com/ansible-collections/kubernetes/blob/master/plugins/lookup/k8s.py#L1-L17

Reply:
Take my comment back after having looked at all of the other modules.